### PR TITLE
[API-1188] Create daily sidekiq job to find, catch and rerun "uploaded" statuses for when SNS fails

### PR DESCRIPTION
**Pull request by templeman15**
## Description of change
Setup a daily sidekiq job to find `UploadSubmissions` that are stuck in the `uploaded` status when SNS fails for some reason. The job runs the `UploadProcessor` worker.

## Original issue(s)
[Create daily sidekiq job to find, catch and rerun "uploaded" statuses for when SNS fails](https://vajira.max.gov/browse/API-1188)

### CODE REVIEW

File: config/sidekiq_scheduler.yml [hunk 1]
```diff
@@ -130,6 +130,11 @@ VBADocuments::ReportUnsuccessfulSubmissions:
   class: VBADocuments::ReportUnsuccessfulSubmissions
   description: "Weekly report of unsuccessful submissions"
 
+VBADocuments::RunUnsuccessfulSubmissions:
+  cron: "0 0 * * MON-FRI America/New_York"
```
Comment (templeman15) [inline config/sidekiq_scheduler.yml:134]: Is this correct for running the job every day? @bastosmichael

Comment (bastosmichael) [reply config/sidekiq_scheduler.yml:134]: Looks good but make sure that the RunUnsuccessfulSubmissions gets triggered before ReportUnsuccessfulSubmissions time wise.

**Review by bastosmichael — CHANGES_REQUESTED** [review]
Minor change to the timeline for when this job should run otherwise looks good.

File: config/sidekiq_scheduler.yml (hunk 1 above)
Comment (templeman15) [reply config/sidekiq_scheduler.yml]: @bastosmichael updated to run daily at 2300 hours.

**Review by bastosmichael — APPROVED** [review]

**Pull request merged**